canvas_modules/common-canvas/__tests__/notification-panel/notification-panel-test.js

```diff
@@ -128,11 +125,14 @@ describe("notification panel renders correctly", () => {
 			</Provider>
 		);
 
-		const notificationPanel = wrapper.find("NotificationPanel");
-		expect(JSON.stringify(notificationPanel.prop("notificationConfig"))).to.equal(JSON.stringify(notificationConfigDefault));
-		expect(JSON.stringify(notificationPanel.prop("messages"))).to.equal(JSON.stringify(notificationMessages));
-		expect(notificationPanel.prop("subPanelData")).to.equal(subPanelData);
-		expect(notificationPanel.prop("closeSubPanel")).to.equal(closePanelFn);
+		expect(getByText("Notifications Panel")).to.exist;
```

Comment:
This is not really testing the props to check if the `notificationConfigDefault`, `notificationMessages`, `subPanelData` objects are set.

Reply:
@srikant-ch5 We are going to remove `enzyme` and `enzyme-adapter-react-17` completely from our codebase. However we have a lot of unit tests to verify props of the component. I did a simple google search and found this article - https://akoskm.com/how-to-test-props-in-react-testing-library

Can you try `jest.mock` and other ways mentioned in the article to test props in RTL?

Reply:
Hi @nmgokhale I tried above solution yesterday but once I mocked the component it was affecting other test cases, will try to find a solution for this.